### PR TITLE
region-grower update

### PR DESCRIPTION
**Pull request by arnaudon**
Update region-grower and dependencies to latest versions.

### CODE REVIEW

**Comment by arnaudon** [comment]
Just waiting for region-grower release.

File: var/spack/repos/builtin/packages/py-morph-tool/package.py
Comment (arnaudon) [inline var/spack/repos/builtin/packages/py-morph-tool/package.py]:
```suggestion
    version('2.5.0', sha256='279123b59797132a2e4f8fd9df391279fce85891dec046bd1206de4c7d4f8e11')
```

File: var/spack/repos/builtin/packages/py-region-grower/package.py
Comment (arnaudon) [inline var/spack/repos/builtin/packages/py-region-grower/package.py]:
```suggestion
    depends_on('py-morph-tool@2.5.0:', type='run')
```

**Review by matz-e — APPROVED** [review]

**Comment by arnaudon** [comment]
I'll merge if it's ok with @GianlucaFicarelli!

**Pull request merged**
